### ISSUE TITLE
[FEATURE_REQUEST] change keycloak auth to be custom url instead of automatically appending /auth

### ISSUE THREAD

**Issue by lgwapnitsky** [Lissy93/dashy#564]
### Is your feature request related to a problem? If so, please describe.

The new version of keycloak does not expose the /auth path by default. When pulling down the newest code, I have to go in and manually edit the file at `src/utils/KeycloakAuth.js` to remove the path. By default, it reads like this:

```
class KeycloakAuth {
  constructor() {
    const { auth } = getAppConfig();
    const { serverUrl, realm, clientId } = auth.keycloak;
    const initOptions = {
      url: `${serverUrl}/auth`, realm, clientId, onLoad: 'login-required',
    };

    this.keycloakClient = Keycloak(initOptions);
  }
```

but I remove the path in the `url` variable to read as such:

```
  constructor() {
    const { auth } = getAppConfig();
    const { serverUrl, realm, clientId } = auth.keycloak;
    const initOptions = {
      url: `${serverUrl}`, realm, clientId, onLoad: 'login-required',
    };

    this.keycloakClient = Keycloak(initOptions);
  }
```

I originally discovered this issue after filing ticket #520 and working through diagnostics

### Describe the solution you'd like

remove the `/auth` path from `KeycloakAuth.js`, and change the instructions to have the user/administrator put in the full path, depending on the version of KC

### Priority

Medium (Would be very useful)

### Is this something you would be keen to implement

Maybe

**Comment by Lissy93** [comment]
Thanks for the detailed outline, reopened and I will get this fixed :)

**Comment by Lissy93** [comment]
Merged in #557 :)

**Comment by TheUdK** [comment]
This change broke existing setups running earlier versions of KC. Maybe you would want to add add a note to the release as well also documentation for the configuration.

If running KC version before 17? and dashy 2.0.6 you need to change appConfig.auth.keycloak.serverUrl from xx.xx.xx to xx.xx.xx/auth
